### PR TITLE
Initial RHEL 10 e8 Profile

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

* Add e8 Control file
* Add RHEL 10 e8 profile

#### Rationale:

Expand RHEL 10 profile set

### CODE REVIEW

File: products/rhel10/profiles/e8.profile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    SMEs:
+        - shaneboulden
+        - tjbutt58
+
+reference: https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-linux-workstations-and-servers
+
+title: 'Australian Cyber Security Centre (ACSC) Essential Eight'
+
+description: |-
+  This profile contains configuration checks for Red Hat Enterprise Linux 9
```
Comment (jan-cerny) [inline products/rhel10/profiles/e8.profile:13]: s/9/10

File: controls/e8.yml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+    -   id: 'hardening'
+        levels:
+            - base
+        title: 'General hardening of {{{ full_name }}}'
+        rules:
+            - var_system_crypto_policy=default_nosha1
+            - configure_crypto_policy
+            - configure_ssh_crypto_policy
+            ### Secure access
```
Comment (jan-cerny) [inline controls/e8.yml:89]: What do you think about creating sub-controls or separate controls for these subsections?

File: controls/e8.yml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    - id: base
+
+controls:
+    -   id: 'patching'
+        levels:
+            - base
+        title: 'Application and operating system patching'
+        rules:
+            - package_talk_removed
+            - package_talk-server_removed
```
Comment (jan-cerny) [inline controls/e8.yml:15]: Packages like talk, ypbind, quagga, squid aren't part of RHEL 10 so I think these rules should be excluded in the RHEL 10 profile.

File: controls/e8.yml (hunk 2 above)
Comment (Mab879) [reply controls/e8.yml:89]: There is no basis in the PDF for this, but we could do this. 

Would you be fine with the sub controls even though the PDF wouldn't support that?

Comment (jan-cerny) [reply controls/e8.yml:89]: Aha OK then I think it would make problems for the automated reference assignement in future.

Comment (jan-cerny) [reply controls/e8.yml:89]: so please keep it as it is now

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**